### PR TITLE
6118 unify transaction list display

### PR DESCRIPTION
**Pull request by emitche**
- Use same code base for transaction grids in main transactions and loans
- Refactor transactions grid to match the formatting of rest of the site

### CODE REVIEW

**Comment by emitche** [comment]
@hooverlunch: I've completed the refactoring. This is ready to be reviewed. You mentioned adding some simple tests as part of this feature, which can happen in this branch or another branch.

File: app/controllers/concerns/transaction_listable.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module TransactionListable
+  extend ActiveSupport::Concern
+
+  def initialize_transactions_grid(project_id=nil)
```
Comment (emitche) [inline app/controllers/concerns/transaction_listable.rb]: Add space around equal.

File: app/views/admin/transactions/_transactions_grid_definition.html.slim [hunk 2]
```diff
@@ -0,0 +1,10 @@
+= define_grid(@transactions_grid) do |g|
+  - g.column name: t('activerecord.attributes.accounting.transaction.qb_id') , attribute: 'qb_id'
```
Comment (emitche) [inline app/views/admin/transactions/_transactions_grid_definition.html.slim]: Remove space around the comma.

File: spec/features/admin/accounting/transaction_flow_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+feature 'transaction flow' do
+  let(:user) { create_admin(root_division) }
+  let!(:transactions) { create_list(:accounting_transaction, 2) }
+
+  before do
+    login_as(user, scope: :user)
+    allow(Accounting::Quickbooks::FetcherBase).to receive(:new).and_return(double(fetch: nil))
+  end
+
+  scenario 'loads properly', js: true do
```
Comment (emitche) [inline spec/features/admin/accounting/transaction_flow_spec.rb]: Do you want this `js: true` here? Not needed for this specific spec.

Comment (hooverlunch) [inline spec/features/admin/accounting/transaction_flow_spec.rb]: Yes please remove.

**Pull request merged**